bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -53,15 +53,12 @@ public boolean isRequired() {
     @Override
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public Type getType() {
-        boolean isRepeatable = minItems != null && maxItems != null;
-        if (wrapData || getDataRef() != null || isRepeatable) {
-            if (isRepeatable) {
-                return Type.ARRAY;
-            } else {
-                return Type.OBJECT;
-            }
+        Boolean isRepeatable = this.isRepeatable();
+        if ((isRepeatable != null) && isRepeatable) {
+            return Type.ARRAY;
```

Comment:
type must be OBJECT for repeatable panel we are not not supporting array type repeatable fields (array in array) via core components authoring.

Reply:
type of repeatable element (field or panel) is always array

Reply:
it should be object according to the spec.

Reply:
it can be array only if array in array use case is to be supported.